sdk/tables/azure-data-tables/azure/data/tables/_base_client.py

```diff
@@ -55,17 +57,12 @@
 )
 from ._sdk_moniker import SDK_MONIKER
 
-if TYPE_CHECKING:
-    from azure.core.credentials import TokenCredential
-    from typing import Literal
-
 _SUPPORTED_API_VERSIONS = ["2019-02-02", "2019-07-07", "2020-12-06"]
 # cspell:disable-next-line
 _DEV_CONN_STRING = "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1" # pylint: disable=line-too-long
 
 
-def get_api_version(kwargs, default):
-    # type: (Dict[str, Any], Literal["2019-02-02"]) -> Literal["2019-02-02"]
+def get_api_version(kwargs: Dict[str, Any], default: Literal["2019-02-02"]) -> Literal["2019-02-02"]:
```

Comment:
```suggestion
def get_api_version(kwargs: Dict[str, Any], default: Literal["2019-02-02"]) -> str:
```
If the user has overridden the api_version, we don't know what the value will be.

Reply:
Updated!

Reply:
Well, mypy complains incompatible types in assignment with the change.